pandas/indexes/base.py

```diff
@@ -432,6 +432,35 @@ def _update_inplace(self, result, **kwargs):
         # guard when called from IndexOpsMixin
         raise TypeError("Index can't be updated inplace")
 
+    def _get_grouper_for_level(self, group_mapper, level):
+        """
+        Get index grouper corresponding to an index level
+
+        Parameters
+        ----------
+        group_mapper: Group mapping function or None
+            Function mapping index values to groups
+        level : int
```

Comment:
just make this ``level=None`` by default and assert it is None for Index. don't put the (Only used phrase)